pq/client-pq-tls13.c

```diff
@@ -123,18 +123,24 @@ int main(int argc, char** argv)
     WOLFSSL_CTX* ctx = NULL;
     WOLFSSL*     ssl = NULL;
 
+    char *cert_file = CERT_FILE;
+
     /* Check for proper calling convention */
-    if (argc != 2) {
-        printf("usage: %s <IPv4 address>\n", argv[0]);
+    if (argc != 2  && argc != 3) {
+        printf("usage: %s <IPv4 address> [<Root cert>]\n", argv[0]);
+        printf("Default Root cert: %s\n", cert_file);
         return 0;
+    } else if (argc == 3) {
+        cert_file = CERT_FILE;
```

Comment:
`argv[2]` is never used in this application.  `cert_file`should be set to `argv[2]` at some point.